### DISCUSSION TITLE
Welcome to pylabel Discussions!

### DISCUSSION THREAD

**Discussion by alexheat** [pylabel-project/pylabel#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iMayuqi** [comment]
why the output type of coco2yolo is yaml not txt.

**Comment by alexheat** [comment]
The output is one yaml and multiple txt files

**Comment by alexheat** [comment]
@MAyuqi1 I think your question may be answered by the documentation here: https://pylabel.readthedocs.io/en/latest/pylabel.html#module-pylabel.exporter 

> A list with 1 or more paths (strings) to annotations files. If a YAML file is created then the first item in the list will be the path to the YAML file.
